example/flutter/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you leave comments on what lines of the file were added/modified from the generated file? That way it will be easier to deal with conflicts etc. if the Flutter project needs to be updated in the future.

Reply:
Oops. I mentioned this file in the above comment. It isn't modified by me. But, other paths are correct. I did a copy paste mistake with `android/src/main/AndroidManifest.xml`.

Anyways, I added it to the correct file :)